### PR TITLE
LP-2295 Fix Linting Errors

### PR DESCRIPTION
**Pull request by abdullah998**
### Description

[LP-2295](https://philanthropyu.atlassian.net/browse/LP-2295)

There was a false positive which I had to disable with # pylint: disable=E1101

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/custom_fields/multiselect_with_other/db/fields.py [hunk 1]
```diff
@@ -116,12 +116,10 @@ def to_python(self, value):
                 return MSFList(choices, list(value))
         return MSFList(choices, [])
 
-    def _check_other_max_length_attribute(self, **kwargs):
+    def _check_other_max_length_attribute(self):
```
Comment (zahid-arbisoft) [inline openedx/features/custom_fields/multiselect_with_other/db/fields.py:119]: At line `# 147` we are using this function and passing `kwargs`, Is removing `**kwargs` enough. Will code work?

```
# Line # 147
errors.extend(self._check_other_max_length_attribute(**kwargs))
```

File: openedx/features/custom_fields/multiselect_with_other/forms/fields.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """This module contains MultiSelect form field with other option"""
-
+# pylint: disable=E1101
```
Comment (zahid-arbisoft) [inline openedx/features/custom_fields/multiselect_with_other/forms/fields.py:2]: 1. Please disables error on the relevant line (i.e. **line # 30**), 
2. Always prefer error-name over error-code i.e.e **no-member** over **E1101**

```
self.error_messages.update(
    dict(invalid_length=_(  # pylint: disable=no-member
        'Other field value, maximum allowed length violation. Allowed limit is upto {other_max_length}'
        ' characters.').format(
        other_max_length=other_max_length)))
```

**Review by zahid-arbisoft — CHANGES_REQUESTED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/custom_fields/multiselect_with_other/db/fields.py (hunk 1 above)
Comment (abdullah998) [reply openedx/features/custom_fields/multiselect_with_other/db/fields.py:119]: Thanks, fixed it now.

File: openedx/features/custom_fields/multiselect_with_other/forms/fields.py (hunk 2 above)
Comment (abdullah998) [reply openedx/features/custom_fields/multiselect_with_other/forms/fields.py]: Corrected it.

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by zahid-arbisoft — APPROVED** [review]

**Review by aliassad — APPROVED** [review]
LGTM 👍

**Pull request merged**